Liroy/bootstrap.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+#Step 1 - Update CentOS 7 system
+echo "Updating system"
+sudo yum -y install epel-release
+sudo yum -y update
+sudo yum install -y yum-utils
+sudo yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo
+
+#Install Docker
+echo "Installing Docker"
+sudo yum -y install docker docker-ce docker-ce-cli containerd.io
+which docker
```

Comment:
Do you know of a more elegant way to validate a command was installed?
Are you concerned yum didn't do the job? What about grabbing yum log/output/error?